src/content/docs/opentelemetry/opentelemetry-introduction.mdx

```diff
@@ -18,23 +18,28 @@ redirects:
+
+To delve deeper into how New Relic processes and ingests OpenTelemetry data, explore the following pages:
+* [OpenTelemetry data in New Relic](/docs/opentelemetry/best-practices/opentelemetry-data-overview/): Learn how New Relic receives, processes, and ingests OpenTelemetry data
+* [New Relic OTLP endpoint](/docs/opentelemetry/best-practices/opentelemetry-otlp/): Configure our OTLP endpoint to send data to New Relic
 
-This page provides an overview of OpenTelemetry and New Relic. For working code examples demonstrating common integration patterns including APM and infrastructure monitoring, see [get started with OpenTelemetry with New Relic](/docs/opentelemetry/get-started/opentelemetry-get-started-intro/). For information on how New Relic receives, processes, and ingests OpenTelemetry data, see [OpenTelemetry data in New Relic](/docs/opentelemetry/best-practices/opentelemetry-data-overview/), and in particular details on [New Relic's OTLP endpoint](/docs/opentelemetry/best-practices/opentelemetry-otlp/).
 
 ## Benefits of OpenTelemetry [#benefits]
 
 OpenTelemetry is a vendor-neutral open standard for instrumenting and exporting telemetry data. The project scope is quite extensive, including:
 
-* A [specification](https://opentelemetry.io/docs/specs/otel/) of language agnostic APIs for instrumenting the core pillars of observability (traces, metrics, logs), and SDKs for configuring how telemetry is exported out of process, with implementations exist in [11+](https://opentelemetry.io/docs/languages/) languages. There is a substantial catalog of instrumentation available built on top of these APIs.
+* A [specification](https://opentelemetry.io/docs/specs/otel/) of language agnostic APIs for instrumenting the core pillars of observability (traces, metrics, logs), and SDKs for configuring how telemetry is exported out of process, with implementations exist in [11+ languages](https://opentelemetry.io/docs/languages/). There is a substantial catalog of instrumentation available built on top of these APIs.
 * [OTLP](https://opentelemetry.io/docs/specs/otlp/), a general-purpose telemetry data delivery protocol.
-* [Semantic conventions](https://opentelemetry.io/docs/specs/semconv/) describing the shape of telemetry data for common domains (i.e. HTTP servers, messaging systems, and many more).
+* [Semantic conventions](https://opentelemetry.io/docs/specs/semconv/) describing the shape of telemetry data for common domains (HTTP servers, messaging systems, and many more).
 * The [collector](https://opentelemetry.io/docs/collector/), a highly configurable and extensible data collection and processing pipeline used to monitor infrastructure and enrich, filter, and otherwise transform telemetry.
```

Comment:
```suggestion
* The [Collector](https://opentelemetry.io/docs/collector/), a highly configurable and extensible data collection and processing pipeline used to monitor infrastructure and enrich, filter, and otherwise transform telemetry.
```